src/util/exportRequestBuilder.ts

```diff
@@ -28,11 +29,18 @@ export interface BuilderRequestQueryParams {
  * Note: currently only supports type parameters and elements.
  */
 export function buildExportRequestString(request: BuilderRequest) {
-  const { baseUrl, exportType, id, queryParams } = request;
+  const { baseUrl, exportType, id, queryParams, customParams } = request;
   const exportPath = buildExportPath(exportType, id);
   const queryString = buildQueryString(queryParams) ?? '';
+  const customQueries = customParams ?? '';
 
-  return baseUrl + exportPath + queryString;
+  if (queryString || customQueries) {
+    if (exportType === 'measure-bundle') {
+      return baseUrl + exportPath + '?' + customQueries;
```

Comment:
```suggestion
      return baseUrl + exportPath + '?' + queryString + customQueries;
```
We still need the query string in this case to add in the `organizeOutputBy` parameter that comes from the switch at the bottom of the page.